component-samples/scripts/keys_gen.sh

```diff
@@ -0,0 +1,247 @@
+#!/bin/bash
+#
+# Copyright 2021 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+shopt -s extglob
+set -e
+bold=$(tput bold)
+normal=$(tput sgr0)
+
+components=("manufacturer" "owner" "reseller")
+COMP_PATH=$1
+CERTS_PATH=$COMP_PATH/creds
+MFG_CERTS_PATH=$COMP_PATH/${components[0]}
+OWN_CERTS_PATH=$COMP_PATH/${components[1]}
+RES_CERTS_PATH=$COMP_PATH/${components[2]}
+
+usage()
+{
+        echo -e "Usage:
+        $0  <absolute_path_of_pri_components>  <OPTIONS>\n
+        OPTIONS:
+                -m => To create manufacturer keystore
+                -o => To create owner keystore
+                -r => To create reseller keystore
+                -t => To create TLS keystore
+                -d => To create device pem files
+                -h => Prints help message\n
+        NOTE: Order of arguments passing shouldn't change"
+}
+
+keystore_gen()
+{
+        PASS_KEY=`openssl rand --base64 12 | tr -dc 0-9A-Za-z`
+        # RSA2048 Keys Generations
+        openssl genrsa -F4 -out $CERTS_PATH/${1}_rsa2048_private.key 2048
```

Comment:
Check if we can redirect the majority of log messages to some file and only print information to track status of the outcome.

Reply:
Updated